extensions/stackdriver/stackdriver.cc

```diff
@@ -70,14 +70,24 @@ int getMonitoringExportInterval() {
   return 60;
 }
 
-// Get logging export interval from node metadata in milliseconds. Returns 60
+// Get logging export interval from node metadata in nanoseconds. Returns 10
 // seconds if interval is not found in metadata.
-int getLoggingExportIntervalMilliseconds() {
+long int getLoggingExportIntervalNanoseconds() {
```

Comment:
Should we get rid of this node metadata override if we are going to control it via filter config?

Reply:
Removed